internal/binding/binding.go

```diff
@@ -335,6 +335,7 @@ func Declarations(paths ...string) ([]Declaration, error) {
 #define __inline
 #define __restrict
 unsigned __builtin_bswap32 (unsigned x);
+unsigned __builtin_bswap16 (unsigned x);
```

Comment:
Out of curiosity, why did this become necessary?

Reply:
Without this code generation fails on my Arch machine. Motivated by your question I searched a bit and it comes from a change in glibc 2.28 that uses this built-in function if GCC 4.8 or later is available.

Reply:
Thanks.